src/extractor/guidance/intersection_handler.cpp

```diff
@@ -375,19 +391,29 @@ std::size_t IntersectionHandler::findObviousTurn(const EdgeID via_edge,
 
     const EdgeData &in_data = node_based_graph.GetEdgeData(via_edge);
     const auto in_classification = in_data.road_classification;
+
+    const auto obvious_by_road_class = [](const RoadClassification in_classification,
```

Comment:
This is a pretty long lambda function that doesn't actually capture any values. Could this be a member function?